text/0000-cargo-libstd-awareness.md

```diff
@@ -0,0 +1,236 @@
+- Feature Name: cargo_stdlib_awareness
+- Start Date: 2015-05-26
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+
+# Summary
+
+Currently, Cargo doesn't know what parts of the standard library packages depend on.
+By giving it this knowledge, we can make cross compilation and exotic platform development easier, simplify rustbuild, and allow anyone to easily specify bounds on the standard library for their packages.
+This will allow building parts of the standard library from source, but in order to not disrupt existing workflows, the binaries that come with rustc will still be used by default.
+
+
+# Motivation
+
+First, some background.
+`rustc` can be given crates and their location with `--extern <name>=<path>`.
```

Comment:
"given crates and their location" -> "told the location of crates" perhaps? The word "given" here seems a bit odd to me.